src/zen/kbs/ZenKeyboardShortcuts.mjs

```diff
@@ -650,6 +650,40 @@ class nsZenKeyboardShortcutsLoader {
         'zen-compact-mode-shortcut-show-toolbar'
       )
     );
+    // Glance mode's KeySet
+    newShortcutList.push(
+      new KeyShortcut(
+        'zen-glance-expand',
+        'Enter',
+        '',
+        '',
+        '',
+        'cmd_zenGlanceExpand',
+        'zen-glance-shortcut-expand'
+      )
+    );
+    newShortcutList.push(
+      new KeyShortcut(
+        'zen-glance-split',
+        'S',
+        '',
+        '',
+        '',
+        'cmd_zenGlanceSplit',
+        'zen_glance-shortcut-split'
+      )
+    );
+    newShortcutList.push(
```

Comment:
Esc is already a shortcut, purposefully done on the content process. I'd do the same with the other shortcuts

Reply:
The ESC key is arleady added, im saying that the other shortcuts should be added there as well.

Reply:
Are you referring to [this line](https://github.com/zen-browser/desktop/blob/48ab77ef5279e856e73ad5a7204d19351a7b6d14/src/browser/base/zen-components/ZenGlanceManager.mjs#L41)? The Esc key is hardcoded in this case.

How would you add the other hotkeys while making the hotkeys customiseable in `ZenGlanceManager.mjs`?